grocery_lists/api.py

```diff
@@ -0,0 +1,70 @@
+from grocery_lists.models import GroceryListCreate, GroceryListItemCreate, GroceryListUpdate
+from grocery_lists.services import add_item_to_grocery_list, create_grocery_list, delete_grocery_list, delete_grocery_list_item, get_grocery_list_items, get_grocery_lists_by_group_id, toggle_grocery_list_item, update_grocery_list
+from ninja import Router
+from django.http import JsonResponse
+
+router = Router()
+
+@router.get("/")
+def get_grocery_lists_handler(request, group_id: int):
+    try:
+        ret = get_grocery_lists_by_group_id(group_id)
+        return JsonResponse(ret, safe=False, status=200)
+    except Exception as e:
+        return JsonResponse({"status": "error", "message": "Error in fetching grocery lists"}, status=400)
+    
+@router.post("/create")
+def add_grocery_list_handler(request, new_grocery_list: GroceryListCreate):
+    try:
+        ret = create_grocery_list(new_grocery_list)
+        return JsonResponse({"id": ret}, status=200)
+    except Exception as e:
+        return JsonResponse({"status": "error", "message": "Error in saving grocery list"}, status=400)
+    
+@router.put("/update")
+def update_grocery_list_handler(request, grocery_list: GroceryListUpdate):
+    try:
+        update_grocery_list(grocery_list)
+        return JsonResponse({"success": True})
```

Comment:
i havent checked how you are handling these on the frontend but since you never return `{"success": False}` you can just return a 204 (indicates that the server has successfully handled the request but has no data to return)